clubs/views.py

```diff
@@ -2,11 +2,13 @@
 from django.core.urlresolvers import reverse
 from django.conf import settings
 from django.core.mail import send_mail
+from django.template import Context
 from rest_framework import serializers, viewsets, permissions
 
 from .models import Club
 from . import email
 
+
```

Comment:
pep convention to leave two blank lines between imports and classes, classes and classes, etc.